examples/eq/sketch.js

```diff
@@ -0,0 +1,132 @@
+/**
+ *  Example: Apply a p5.EQ filter to a p5.Noise.
+ *  Visualize the sound with FFT.
+ *  Use control points to change the spline that shapes the soundwave
+ */
+
+var fft, noise, eq;
+var eqSize;
+
+var cntrlPts = []; 
+var ctrlPtRad, cntrlIndex;
+var splineVerts = [];
+var pressed; 
+var description;
+
+function setup() {
+  createCanvas(710, 256);
+  //sound wave color
+  fill(255, 40, 255);
+  ctrlPtRad = 15;
+  
+  eqSize = 8;
+  eq = new p5.EQ(eqSize);
+
+
+  // Disconnect noiose generator from master output.
+  // Then, connect it to the EQ, so that we only hear the EQ'd sound
+  noise = new p5.Noise();
+  noise.disconnect();
+  eq.process(noise);
+  noise.start();
+
+  //use an fft to visualize the sound
+  fft = new p5.FFT();
+
+  for (var i = 0; i < eqSize; i++) {
+    cntrlPts[i] = new CntrlPt(i, (width/(eqSize-1)) * i, height/2);
+    eq.bands[i].freq(map(cntrlPts[i].x, 0, width, 160, 20480));
```

Comment:
I think this would be a stronger example if you used the default EQ positions—highlight the exponential scaling that you worked on.

Some nice to haves
- Labels for the nodes showing their frequency and gain
- It'd be really cool to scale the frequency spectrum logarithmically. There's so much good stuff at the lower end of the frequency spectrum, every octave gets equal number of pixels on the screen that gives the user more room to play with meaningful frequencies. [Here's](https://therewasaguy.github.io/p5-music-viz/demos/04_fft_freq_spectrum/) an attempt I made at log/lin toggle for FFT visualization, in case it's helpful